spec/services/quarterly_message_integration_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+require "business_time"
+
+# I really hate putting this test here as it does not correspond
```

Comment:
oh yes totally fine with me to create seed data in feature specs. If you look at our existing feature specs, we are doing just that: https://github.com/18F/dolores-landingham-bot/blob/master/spec/features/edit_scheduled_messages_spec.rb

Now that I rethink this, though, it seems like what we're really testing here is that `DailyMessageSender.new.run` is doing its job. 

Hard to test clockwork job since that is not a feature with a UI. So maybe capybaras is not the anwswer.

Thoughts on moving all this to `DailyMessageSender.`  unit tests? I do like your usage of Timecop here...